src/main/reservation/main/G1HRPS/ReservationManager.java

```diff
@@ -53,21 +70,28 @@ public void SaveDB() {
 	 * @param new_status
 	 */
 	public boolean SetReservationStatus(Reservation reservation, ReservationStatus new_status) {
-		// TODO - implement ReservationManager.SetReservationStatus
```

Comment:
To add ReservationManager.CheckIn and ReservationManager.CheckOut functions instead.